### PR TITLE
fix: ProgressTracker Examples crash & layout

### PR DESCRIPTION
**Pull request by kazaky**
## Changes
- Fix crash in ProgressTracker Examples
- Also fixed layout not showing all intents

## 🤔 Context
Closes #1146

### CODE REVIEW

**Comment by kazaky** [comment]
@coderabbitai summary

File: catalog/src/main/kotlin/com/adevinta/spark/catalog/examples/samples/progresstracker/ProgressTrackerExamples.kt [hunk 1]
```diff
@@ -220,9 +223,9 @@ private fun ColumnScope.ProgressTrackerDisabled() {
 private fun ColumnScope.ProgressTrackerSizes() {
     val selectedStep by remember { mutableIntStateOf(1) }
     val items = persistentListOf(
-        ProgressStep("", true),
-        ProgressStep("", true),
-        ProgressStep("", false),
+        ProgressStep("a", true),
```
Comment (soulcramer) [inline catalog/src/main/kotlin/com/adevinta/spark/catalog/examples/samples/progresstracker/ProgressTrackerExamples.kt:226]: note that the horizontal ProgressTrackerRow doesn't require a label

File: catalog/src/main/kotlin/com/adevinta/spark/catalog/examples/samples/progresstracker/ProgressTrackerExamples.kt [hunk 2]
```diff
@@ -249,11 +252,13 @@ private fun ColumnScope.ProgressTrackerSizes() {
...
 @Preview
 private fun ColumnScope.ProgressTrackerColors() {
+    val hScrollState = rememberScrollState()
+
     val selectedStep by remember { mutableIntStateOf(1) }
     val items = persistentListOf(
-        ProgressStep("", true),
-        ProgressStep("", true),
-        ProgressStep("", false),
+        ProgressStep("Step 1", true),
```
Comment (soulcramer) [inline catalog/src/main/kotlin/com/adevinta/spark/catalog/examples/samples/progresstracker/ProgressTrackerExamples.kt:259]: note that the horizontal ProgressTrackerRow doesn't require a label

**Review by soulcramer — APPROVED** [review]

**Comment by SimonMarquis** [comment]
@kazaky you don't need to keep the entire default PR body if it is unchanged ;)

**Pull request merged**